phys2bids/interfaces/txt.py

```diff
@@ -54,10 +54,16 @@ def populate_phys_input(filename, chtrig):
     for item in range_list:
         units.append(item.split(' ')[1])
     # get names
-    header[4][1:]
-    names = ['time']+header[4][1:]
+    orig_names=header[4][1:]
+    names = ['time',orig_names[chtrig]]
```

Comment:
Can you call it literally `'trigger'`?

Reply:
Done and also revised the docstring identation